kuma/static/js/auth.js

```diff
@@ -54,71 +54,75 @@
         Show notifications about account association status as part of the
         registration process.
     */
-    if(win.mdn.features.localStorage) (function() {
-        try {
-            var $browserRegister = $('#browser_register');
-            var matchKey = 'account-match-for';
-            var matchCurrent = $browserRegister.data(matchKey);
-            var matchStored = localStorage.getItem(matchKey);
-
-            // The user is on the registration page and has been notified that
-            // there is an MDN profile with a matching email address.
-            if(matchCurrent && !matchStored) {
-                localStorage.setItem(matchKey, matchCurrent);
-            }
+    if(win.mdn.features.localStorage) {
+        (function() {
+            try {
+                var $browserRegister = $('#browser_register');
+                var matchKey = 'account-match-for';
+                var matchCurrent = $browserRegister.data(matchKey);
+                var matchStored = localStorage.getItem(matchKey);
+
+                // The user is on the registration page and has been notified that
+                // there is an MDN profile with a matching email address.
```

Comment:
Nit: For multi line comments we should use the `/** The name of the module. */` JSDoc standard. This will then also match how comments are done on Bedrock.

Reply:
I didn't write or edit the comment, not sure why it shows up as a change. I'd like to leave it alone and just make the linter happy.